pkg/controller/performanceprofile/components/profile/profile_test.go

```diff
@@ -88,6 +88,24 @@ var _ = Describe("PerformanceProfile", func() {
 			Expect(err.Error()).To(ContainSubstring(fmt.Sprintf("the page size should be equal to %q or %q", hugepagesSize1G, hugepagesSize2M)))
 		})
 
+		It("should allow cpus allocation with no reserved CPUs", func() {
+			reservedCPUs := v1.CPUSet("")
+			isolatedCPUs := v1.CPUSet("0-7")
+			profile.Spec.CPU.Reserved = &reservedCPUs
+			profile.Spec.CPU.Isolated = &isolatedCPUs
+			err := ValidateParameters(profile)
+			Expect(err).Should(Not(HaveOccurred()))
+		})
+
+		It("should allow cpus allocation with no isolated CPUs", func() {
```

Comment:
We have strict validation regarding isolated CPUs, https://github.com/openshift-kni/performance-addon-operators/blob/af31700a496376c6cf70508ba0e65289479bc127/pkg/controller/performanceprofile/components/profile/profile.go#L23

Reply:
sure, adding the field with empty set is legal, isn't it?

Reply:
Yep, it should work, but it definitely wasn't an intention :)

Reply:
Fair enough, so in this PR I think we should either add tests to check this behaviour (because we allow it) or pivot the PR and add a validation to reject a setting with no isolated CPUs. Either is fine for me. Does this plan make sense?

Reply:
cc @MarSik

Reply:
totally makes sense, +1 to add vaalidation